pkg/flowaggregator/flowaggregator_test.go

```diff
@@ -278,3 +282,53 @@ func createElement(name string, enterpriseID uint32) ipfixentities.InfoElementWi
 	ieWithValue, _ := ipfixentities.DecodeAndCreateInfoElementWithValue(element, nil)
 	return ieWithValue
 }
+
+func TestFlowAggregator_updateFlowAggregator(t *testing.T) {
```

Comment:
Since watchConfiguration has infinite loop, which is difficult to implement unit test, I break down the updateFlowAggregator part for adding unit test.
flowRecordExpiryCheck also has same issue. I think if want to add unit test for flowRecordExpiryCheck, we need to break it down to several functions for testing.

Reply:
I modified the test for `watchConfiguration `. In that test, I tried to mimic the replacement (Remove and Create) and see if I can receive the correct message from updateCh.